packages/bpk-component-card/src/BpkCardWrapper.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import { borderSizeLg } from '@skyscanner/bpk-foundations-web/tokens/base.es6';
+
+import { cssModules } from '../../bpk-react-utils';
+
+import STYLES from './BpkCardWrapper.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  card: Node,
+  className: ?string,
+  backgroundColor: string,
+  header: Node,
+};
+
+const BpkCardWrapper = (props: Props) => {
+  const { backgroundColor, card, className, header } = props;
+
+  const classNames = getClassName('bpk-card-wrapper', className);
+
+  return (
+    <div
+      className={classNames}
+      style={{
+        boxShadow: `0 0 0 2px ${backgroundColor}`,
+        '--background-color': backgroundColor,
+      }}
+    >
+      <section
```

Comment:
what's the reason we're using `section` rather than `div` here (and below for the card as well)?

Reply:
I think the [section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)  represents a standalone section and might be semantic, and too much `div` looks not very good. Anyway the `div` also works here.

Reply:
The reason I'm asking is because looking at the Mozilla definition it doesn't seem like it could apply here so wasn't sure if there was a specific reason to use it 🙂 
```The <section> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.```